src/components/classic/documentation/testing-changes-locally.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default_md
+title: Testing changes locally 
+title-class: page-title-classic
+type: classic
+---
+
+[Developers](developers) > [Testing changes locally](testing-changes-locally)
+
+Testing changes locally
+-----------------------------
+
+### Broker
+After running existing or newly added unit tests, if you want to do integration testing on your change, you can [build](building) the repo locally to generate the release bundle and then unzip the bundle to test broker behaviour. 
+```
+# At root directory
+$ mvn -Dtest=false -DfailIfNoTests=false clean install 
+```
+If the build was successful. You will see a message similar to the one below
+```
+[INFO] Installing /Users/xxx/amzn/personal_workspace/activemq/assembly/target/apache-activemq-6.2.0-SNAPSHOT-bin.zip to /Users/xxx/.m2/repository/org/apache/activemq/apache-activemq/6.2.0-SNAPSHOT/apache-activemq-6.2.0-SNAPSHOT-bin.zip
```

Comment:
I think either this can use far more generalised paths, e.g <home-dir> prefix, or /path/to/.....or better yet, it would not be here at all.

The typical place to point to Maven build output is a module `target` directory, so I'd suggest telling them to wait for e.g the completion of the build (e.g 'BUILD SUCCESS') and then go into e.g `assembly/target` to find the built content to then use.